docs/partials/_video-recording-supported-browsers.mdx

```diff
@@ -0,0 +1,5 @@
+:::caution
+
+Currently, Cypress only supports video recording for [supported Chromium-based browsers](/guides/guides/launching-browsers#Browsers) (Chrome/Electron/Edge). Support for Firefox can be tracked in [this issue](https://github.com/cypress-io/cypress/issues/18415).
```

Comment:
Tiny tweaks to match similar callouts with the "!" icon. 

```suggestion
&#8239;<Icon name="exclamation-triangle" /> Currently, Cypress supports video recording for [supported Chromium-based browsers](/guides/guides/launching-browsers#Browsers) (Chrome/Electron/Edge). Support for Firefox can be tracked in [this issue](https://github.com/cypress-io/cypress/issues/18415).

```

Reply:
ah thanks, i tried that but saw the link didn't work. Didn't remember what the hack was